styles/pymakr.less

```diff
@@ -1035,5 +1025,352 @@ div.pymakr-feedback {
 }
 .xterm-rows {
   color: @text-color-highlight !important;
-  background-color: @base-border-color !important;
+  background-color: transparent !important;
+}
+
+.chrome-tabs .chrome-tab {
```

Comment:
do we need to use this chrome-tabs css code, I saw that css file should be already included byt chrome-tabs library

Reply:
It's included but it's not being used. Only the logical part is being used.

Reply:
if it is not used then we shouldn't include it

Reply:
Should I delete the style file from the chrome-tabs fork?

Reply:
The library's style are not being used but these class definitions (from pymakr.less) are. All of them.

Reply:
Ideally, it would be great to figure out how to use the styles from your chrome-tabs fork.
Please spend a bit of time to see if that isn't somehow possible.

Also, the first file of this PR css/chrome-tabs.css contains the same code as the code included in this file.
do we need to include that code as well?

Reply:
My bad. I forgot to remove the /css folder. 

Atom automatically injects all the style files (.less and .css) from `styles` folder. I'm using some atom `less` variables (to match with the current theme) in the tab's classes, so we can't separate them. I know it looks something modularized or encapsulated, maybe because I kept the class structure and even the names. I did that because I didn't want to re-arrange the tabs structure, so I could save some time. The style doesn't have any relation with the library.

<img width="477" alt="image" src="https://user-images.githubusercontent.com/15652671/74668098-cdf39300-51a4-11ea-8fd9-d9920bbd7b30.png">

As you can see in the picture above, the selectors are all defined. I just changed the layout to match Pymakr's and kept the classes names and element structure:

<img width="372" alt="image" src="https://user-images.githubusercontent.com/15652671/74668417-743f9880-51a5-11ea-9109-9ee785b998c9.png">

What I think I could do is create a separated .less file, just for the tabs. And also change the class names maybe.

Reply:
OK thank you for your explanation. And that deleted folder.

I think we should put this new code into a new less file. I saw that that pymakr.less file is around 1.5K lines. I think we shouldn't put any new code there.

Reply:
You're right. Better start to create separated style files, specially before starting the Pybytes integration.